utils/models.py

```
@@ -0,0 +1,178 @@
+            self.tokenizer.model_max_length
+            if model_max_length is None
+            else model_max_length
+        )
+        self.name = type(self).__name__
+        self.supports_center_leaning = supports_center_leaning
+
+    @abstractmethod
+    def predict(self, article_body: str, truncate_tokens: bool) -> Leaning:
+        pass
+
+    def get_tokens(
```

Ruff

Missing return type annotation for public function `get_tokens`

(ANN201)